<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Calculator</class>
 <widget class="QMainWindow" name="Calculator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>288</width>
    <height>273</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Calculator</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="0">
     <widget class="QPushButton" name="Btn6">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #6200EE;
	border: 1px rgb(0, 0, 0);
	color: rgb(255, 255, 255);
	padding: 5px;
}

QPushButton:pressed {
	background-color: #3700B3;
	border: 1px rgb(0, 0, 0);
	color: rgb(255, 255, 255);
	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>6</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="Btn8">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #6200EE;
	border: 1px rgb(0, 0, 0);
	color: rgb(255, 255, 255);
	padding: 5px;
}

QPushButton:pressed {
	background-color: #3700B3;
	border: 1px rgb(0, 0, 0);
	color: rgb(255, 255, 255);
	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>8</string>
      </property>
     </widget>
    </item>
    <item row="3" column="4">
     <widget class="QPushButton" name="BtnSub">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #6200EE;
	border: 1px rgb(0, 0, 0);
	color: rgb(255, 255, 255);
	padding: 5px;
}

QPushButton:pressed {
	background-color: #3700B3;
	border: 1px rgb(0, 0, 0);
	color: rgb(255, 255, 255);
	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>-</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QPushButton" name="Btn2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #6200EE;
	border: 1px rgb(0, 0, 0);
	color: rgb(255, 255, 255);
	padding: 5px;
}

QPushButton:pressed {
	background-color: #3700B3;
	border: 1px rgb(0, 0, 0);
	color: rgb(255, 255, 255);
	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>2</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QPushButton" name="Btn1">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #6200EE;
	border: 1px rgb(0, 0, 0);
	color: rgb(255, 255, 255);
	padding: 5px;
}

QPushButton:pressed {
	background-color: #3700B3;
	border: 1px rgb(0, 0, 0);
	color: rgb(255, 255, 255);
	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>1</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QPushButton" name="Btn9">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #6200EE;
	border: 1px rgb(0, 0, 0);
	padding: 5px;
 	color: rgb(255, 255, 255);
}

QPushButton:pressed {
	background-color: #3700B3;
	border: 1px rgb(0, 0, 0);
	color: rgb(255, 255, 255);
	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>9</string>
      </property>
     </widget>
    </item>
    <item row="1" column="4">
     <widget class="QPushButton" name="BtnAC">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #EF5350;
	border: 1px rgb(0, 0, 0);
	padding: 5px;
}

QPushButton:pressed {
	background-color: #B71C1C;
	border: 1px rgb(0, 0, 0);
	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>AC</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="Btn5">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #6200EE;
	border: 1px rgb(0, 0, 0);
	color: rgb(255, 255, 255);
	padding: 5px;
}

QPushButton:pressed {
	background-color: #3700B3;
	border: 1px rgb(0, 0, 0);
	color: rgb(255, 255, 255);
	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>5</string>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QPushButton" name="BtnDel">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #EF5350;
	border: 1px rgb(0, 0, 0);
	padding: 5px;
}

QPushButton:pressed {
	background-color: #B71C1C;
	border: 1px rgb(0, 0, 0);
	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>DEL</string>
      </property>
     </widget>
    </item>
    <item row="2" column="4">
     <widget class="QPushButton" name="BtnDiv">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #6200EE;
	border: 1px rgb(0, 0, 0);
	color: rgb(255, 255, 255);
	padding: 5px;
}

QPushButton:pressed {
	background-color: #3700B3;
	border: 1px rgb(0, 0, 0);
	color: rgb(255, 255, 255);
	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>/</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QPushButton" name="Btn3">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #6200EE;
	border: 1px rgb(0, 0, 0);
	color: rgb(255, 255, 255);
	padding: 5px;
}

QPushButton:pressed {
	background-color: #3700B3;
	border: 1px rgb(0, 0, 0);
	color: rgb(255, 255, 255);
	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>3</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="Btn7">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #6200EE;
	border: 1px rgb(0, 0, 0);
	color: rgb(255, 255, 255);
	padding: 5px;
}

QPushButton:pressed {
	background-color: #3700B3;
	border: 1px rgb(0, 0, 0);
	color: rgb(255, 255, 255);
	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>7</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QPushButton" name="Btn4">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #6200EE;
	border: 1px rgb(0, 0, 0);
	color: rgb(255, 255, 255);
	padding: 5px;
}

QPushButton:pressed {
	background-color: #3700B3;
	border: 1px rgb(0, 0, 0);
	color: rgb(255, 255, 255);
	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>4</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QPushButton" name="BtnMul">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #6200EE;
	border: 1px rgb(0, 0, 0);
	color: rgb(255, 255, 255);
	padding: 5px;
}

QPushButton:pressed {
	background-color: #3700B3;
	border: 1px rgb(0, 0, 0);
	color: rgb(255, 255, 255);
	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>*</string>
      </property>
     </widget>
    </item>
    <item row="3" column="3">
     <widget class="QPushButton" name="BtnAdd">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #6200EE;
	border: 1px rgb(0, 0, 0);
	color: rgb(255, 255, 255);
	padding: 5px;
}

QPushButton:pressed {
	background-color: #3700B3;
	border: 1px rgb(0, 0, 0);
	color: rgb(255, 255, 255);
	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>+</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="5">
     <widget class="QLineEdit" name="Display">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
	background-color: #03DAC6;
	border: 1px rgb(0, 0, 0);
}
</string>
      </property>
      <property name="text">
       <string>0.0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QPushButton" name="Btn0">
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #6200EE;
	border: 1px rgb(0, 0, 0);
	color: rgb(255, 255, 255);
	padding: 5px;
}

QPushButton:pressed {
	background-color: #3700B3;
	border: 1px rgb(0, 0, 0);
	color: rgb(255, 255, 255);
	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QPushButton" name="BtnDecimal">
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #6200EE;
	border: 1px rgb(0, 0, 0);
	color: rgb(255, 255, 255);
	padding: 5px;
}

QPushButton:pressed {
	background-color: #3700B3;
	border: 1px rgb(0, 0, 0);
	color: rgb(255, 255, 255);
	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>.</string>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QPushButton" name="BtnExp">
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #6200EE;
	border: 1px rgb(0, 0, 0);
	color: rgb(255, 255, 255);
	padding: 5px;
}

QPushButton:pressed {
	background-color: #3700B3;
	border: 1px rgb(0, 0, 0);
	color: rgb(255, 255, 255);
	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>EXP</string>
      </property>
     </widget>
    </item>
    <item row="4" column="3">
     <widget class="QPushButton" name="BtnAns">
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #6200EE;
	border: 1px rgb(0, 0, 0);
	color: rgb(255, 255, 255);
	padding: 5px;
}

QPushButton:pressed {
	background-color: #3700B3;
	border: 1px rgb(0, 0, 0);
	color: rgb(255, 255, 255);
	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>Ans</string>
      </property>
     </widget>
    </item>
    <item row="4" column="4">
     <widget class="QPushButton" name="BtnEql">
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #6200EE;
	border: 1px rgb(0, 0, 0);
	color: rgb(255, 255, 255);
	padding: 5px;
}

QPushButton:pressed {
	background-color: #3700B3;
	border: 1px rgb(0, 0, 0);
	color: rgb(255, 255, 255);
	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>=</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>288</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
